src/shim/umq/host_queue.h

```diff
@@ -162,6 +162,8 @@ typedef struct
   uint64_t data_address;
   
   // TODO Ready signal?
+  // padding to 64Bytes cache line
```

Comment:
I thought we need to make sure read_index and write_index are in different cache line, no?